### PR TITLE
[Personal Cabinet] - Admin - Display parent/child list

### PR DESCRIPTION
**Pull request by perepichai**
Task https://github.com/ita-social-projects/OoS-Frontend/issues/1105

### CODE REVIEW

File: src/app/shared/enum/tech-admin/tabs/users.ts
Comment (litvinets) [inline src/app/shared/enum/tech-admin/tabs/users.ts:5]: maybe you can move it to role.ts and add it to enum Role?

File: src/app/shared/utils/utils.ts
Comment (litvinets) [inline src/app/shared/utils/utils.ts:124]: let's have constant noInformation='не вказано'

File: src/app/shell/admin-tools/admin-tools-routing.module.ts
Comment (litvinets) [inline src/app/shell/admin-tools/admin-tools-routing.module.ts:20]: use one path users/:index
and pass empty string for example if there is no param

File: src/app/shell/admin-tools/users/users.component.ts
Comment (litvinets) [inline src/app/shell/admin-tools/users/users.component.ts:30]: move readonly to the very begin as in the rest of the files:)

File: src/app/shell/admin-tools/users/users.component.ts
Comment (litvinets) [inline src/app/shell/admin-tools/users/users.component.ts:37]: add Typo

File: src/app/shell/admin-tools/users/users.component.ts
Comment (litvinets) [inline src/app/shell/admin-tools/users/users.component.ts:48]: I think it's not possible to have !val not an empty string (I mean is it possible to receive here null/undefined/0/NaN ??? ), right? So you can simplify it to
this.filterValue = val;

**Review by dmitryy90 — APPROVED** [review]

**Review by litvinets — APPROVED** [review]
Fix please comments and DO NOT merge for now. let's wait until this PR will be merged  https://github.com/ita-social-projects/OoS-Frontend/pull/1181

**Pull request merged**
